doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -5630,7 +5630,16 @@ NCSDK-6832: SMP Server sample fails upon initialization
 Other issues
 ************
 
-.. rst-class:: v2-9-0-nRF54H20-1-rc3
+.. rst-class:: v2-9-0-nRF54H20-1
+
+HM-25973: SysCtrl does not always wake up when interrupts are triggered by applications running on local domains.
```

Comment:
@meijemac @shanthanordic fyi

Reply:
@meijemac @shanthanordic Github is not showing the right page, however please review the entry on line 5638 in known_issues.rst